Gemfile.lock

```diff
@@ -259,4 +259,4 @@ DEPENDENCIES
   xcpretty-travis-formatter!
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```

Comment:
@jleandroperez I see we are moving the Bundler 2.0.1 in this PR. Is that required to fix the builds? Also, should we be using the [latest Bundler release v2.0.2](https://github.com/bundler/bundler/releases/tag/v2.0.2)?

/cc @loremattei in case there are some implications that we should consider....